src/RoadRegistry.BackOffice/Uploads/RoadSegmentLaneChangeDbaseRecordsValidator.cs

```diff
@@ -111,12 +107,43 @@ public ZipArchiveProblems Validate(ZipArchiveEntry entry, IDbaseRecordEnumerator
                         }
                         moved = records.MoveNext();
                     }
+
+                    foreach (var attributeRecordsOfIdentifier in allAttributeRecords.GroupBy(record => record.Identifier))
+                    {
+                        var attributeRecords = attributeRecordsOfIdentifier.ToArray();
+                        if (attributeRecords.Length > 2)
+                        {
+                            var recordContext = entry.AtDbaseRecord(attributeRecords[0].Number);
+                            var takenBy = attributeRecords
+                                .Skip(1)
+                                .Select(attributeRecord => attributeRecord.Number)
+                                .ToArray();
+                            problems += recordContext.IdentifierNotUnique(
+                                attributeRecords[0].Identifier,
+                                takenBy
+                            );
+                        }
+                        else if (attributeRecords.Length == 2)
```

Comment:
If exactly 2 records than only an added + removed pair is acceptable (because it's an implicit modify).